library/src/@types/index.d.ts

```diff
@@ -0,0 +1,8 @@
+import { Browser } from 'puppeteer'
```

Comment:
Actually, because there are no tests in the library project at the moment, this file is not needed at all. If later some types are needed for tests, then they can be added to `test/@types`. So I'll delete this file for now.